docs/pages/reference/networking.mdx

```diff
@@ -232,12 +245,8 @@ traffic:
 | - | - | - |
 | 3022 | SSH Service | Incoming SSH connections.|
 | 3026 | Kubernetes Service | HTTPS traffic to a Kubernetes API server.| 
-| 3036 | Database Service | Traffic to MySQL databases.|
-| 5432 | Database Service | Traffic to Postgres databases.|
-| 27017 | Database Service | Traffic to MongoDB instances.|
-| 6379 | Database Service | Traffic to Redis instances.|
 | 3028 | Windows Desktop Service | Teleport Desktop Protocol traffic from Teleport clients.|
 
-Applications registered with the Teleport Application Service can only be
-accessed via the Teleport Proxy Service, not directly via the Application
-Service.
+You can only access enrolled applications and databases through the Teleport Proxy Service.
+The Teleport Application Service and Teleport Database Service leverage the reverse tunnel
+connections through the Teleport Proxy service and cannot expose ports directly.
```

Comment:
```suggestion
The Teleport Application Service and Teleport Database Service use reverse tunnel
connections through the Teleport Proxy Service and cannot expose ports directly.
```
- "use" instead of "leverage" for readability
- Fixing "Proxy Service" capitalization